soroban-env-macros/src/call_macro_with_all_host_functions.rs

```diff
@@ -93,9 +93,15 @@ pub fn generate(file_lit: LitStr) -> Result<TokenStream, Error> {
 
             let r#return = format_ident!("{}", &f.r#return);
 
-            quote! {
-                #[doc = #docs]
-                { #export, #min_proto, #max_proto, fn #name(#(#args),*) -> #r#return }
+            if docs.is_empty() {
```

Comment:
Would also be fine to generate a compiler error if docs is empty with the `compiler_error!` macro, with a note telling the dev to add a doc. Host fns should have docs.

Reply:
Yeah that makes sense. I didn't want to update env.json in this PR so went with this instead. I can open an issue if you think it's worth it.

Reply:
I think this is fine.